app/src/androidTest/java/org/simple/clinic/drugs/search/DrugRepositoryAndroidTest.kt

```diff
@@ -52,4 +53,38 @@ class DrugRepositoryAndroidTest {
 
     assertThat(savedDrugs).isEqualTo(drugs)
   }
+
+  @Test
+  fun search_drugs_should_work_correctly() {
+    // given
+    val searchQuery = "Amlod"
+
+    val amlodipine10 = TestData.drug(
+        id = UUID.fromString("fd422777-d6dd-4afc-9fd1-8147e8578ca4"),
+        name = "Amlodipine",
+        dosage = "10mg"
+    )
+    val amlodipine20 = TestData.drug(
+        id = UUID.fromString("d48fd21f-41fd-4e37-9616-fa989964b945"),
+        name = "Amlodipine",
+        dosage = "20mg"
+    )
+    val losartan25 = TestData.drug(
+        id = UUID.fromString("5db739bb-1fa6-470c-b230-4fead4e92543"),
+        name = "Losartan",
+        dosage = "25mg"
+    )
+    val drugs = listOf(amlodipine10, amlodipine20, losartan25)
+
+    drugRepository.save(drugs).blockingAwait()
+
+    // when
+    val expectedSearchResults = PagingTestCase(pagingSource = drugRepository.search(searchQuery),
```

Comment:
Can we pick the search query such that the case-insensitivity of the SQL query is also verified?